lib/modules/manager/poetry/types.ts

```diff
@@ -27,6 +28,10 @@ export interface PoetrySource {
   url?: string;
 }
 
+export interface PoetryGroup {
```

Comment:
It would be better not to have both a Class and an interface both called `PoetryGroup`